src/app/containers/Timestamp/index.jsx

```diff
@@ -20,6 +20,16 @@ const isToday = timestamp => {
 
 const formatType = timestamp => (isToday(timestamp) ? dateAndTime : dateOnly);
 
+const isSecondTimeStampRequired = (firstPublished, lastPublished) => {
+  if (lastPublished !== firstPublished) {
+    return !(
```

Comment:
This double `!` logic isnt the easiest to glance over, can it be reorganised to be clearer?

Reply:
```
  if (lastPublished === firstPublished) {
    return false;
  }
  return !(
    !isToday(lastPublished) &&
    moment(lastPublished).isSame(firstPublished, 'day')
  );
```
better?